src/components/DetailSidebar/DetailSidebarFundButton.tsx

```diff
@@ -21,21 +22,21 @@ const StyledFundStrong = styled('strong')({
   marginRight: 3,
 });
 
-interface Props {
-  to: string;
-}
-
 /* eslint-disable react/jsx-no-bind */
-const DetailSidebarFundButton: React.FC<Props> = ({ to }) => {
-  const preventDefault = (event: MouseEvent<HTMLButtonElement>) => event.preventDefault();
+const DetailSidebarFundButton: React.FC = () => {
+  const detailContext = useContext(DetailContext);
+
+  const { packageMeta } = detailContext;
+
+  if (!packageMeta?.latest?.funding) {
+    return null;
+  }
+
+  const to = packageMeta?.latest?.funding.url;
```

Comment:
Can this be merged / used with the previous line? Or reuse some variable.

Reply:
I don't get it. I need to return null if funding is no there.  Am I missing something?

Reply:
The `to` uses the same code but with `.url` at the end. Duplicate code imo.

Also what if `.url` is not defined / set?